cfme/tests/control/test_bugs.py

```diff
@@ -192,6 +192,41 @@ def action_for_testing(appliance):
     action_.delete()
 
 
+@pytest.fixture
+def compliance_condition(appliance):
+    expression = (
+        "fill_field(VM and Instance : Name, =, cu-24x7); "
```

Comment:
We can't make this name assumption here, it needs to be pulled from the yamls. The fixture should be dependent on appliance and provider.